tests/Policy_1.py

```diff
@@ -22,11 +22,15 @@
 #
 
 import unittest
+import os.path
 
 import lsst.utils.tests
 from lsst.pex.policy import Policy, NameNotFound
 import lsst.pex.exceptions
 
+
+proddir = lsst.utils.getPackageDir('pex_policy')
```

Comment:
Ok. This was code I copied from another test file (added in commit 2b0dcf74481ab92282073580edaa45795d11f955 so I was technically being consistent). 😄